test/test_quantized_tensor.py

```diff
@@ -172,10 +174,40 @@ def test_qtensor_permute(self):
         # compare dequantized result
         self.assertTrue(np.array_equal(qr1.dequantize().numpy(), qr2.dequantize().numpy()))
         # compare permuted + dequantized result with original transposed result
-        self.assertTrue(np.allclose(qr2.dequantize().numpy(), r.numpy().T, atol=2 / scale))
+        self.assertTrue(np.allclose(qr2.dequantize().numpy(), r.numpy().transpose([1, 0, 2, 3]), atol=2 / scale))
         # make permuted result contiguous
         self.assertTrue(torch.equal(qr2.contiguous().int_repr(), qr2.int_repr()))
 
+        # change memory format
+        qlast = qr.contiguous(memory_format=torch.channels_last)
+        self.assertEqual(qr.stride(), list(reversed(sorted(qr.stride()))))
+        self.assertNotEqual(qlast.stride(), list(reversed(sorted(qlast.stride()))))
+        self.assertTrue(torch.equal(qr.int_repr(), qlast.int_repr()))
```

Comment:
FYI: int_repr() will do contiguous right now, but I think it should preserve the strides,  will fix in https://github.com/pytorch/pytorch/pull/25429 when I have some time.